ocs_ci/ocs/perfresult.py

```diff
@@ -56,17 +53,15 @@ def es_connect(self):
         try:
             self.es = Elasticsearch([{"host": self.server, "port": self.port}])
         except ESExp.ConnectionError:
-            log.error(
+            log.warning(
                 "can not connect to ES server {}:{}".format(self.server, self.port)
             )
-            raise
 
         # Testing the connection to the elastic-search
         if not self.es.ping():
-            log.error(
+            log.warning(
                 "can not connect to ES server {}:{}".format(self.server, self.port)
```

Comment:
Same spelling fix here : Cannot instead of Can not

Reply:
fixed it